test/new_relic/agent/monitors/synthetics_monitor_test.rb

```diff
@@ -78,6 +78,72 @@ def test_records_synthetics_state_from_header
       end
     end
 
+    def test_records_synthetics_info_header_if_available
+      key = SyntheticsMonitor::SYNTHETICS_HEADER_KEY
+      synthetics_payload = [VERSION_ID] + STANDARD_DATA
+      info_payload = <<~PAYLOAD
+        {
+          "version": "1",
+          "type": "automatedTest",
+          "initiator": "cli",
+          "attributes": {
+            "attribute1": "one"
+          }
+        }
+      PAYLOAD
+
+      expected_info = {
+        'version' => '1',
+        'type' => 'automatedTest',
+        'initiator' => 'cli',
+        'attributes' => {
+          'attribute1' => 'one'
+        }
+      }
+
+      with_synthetics_headers(synthetics_payload, headers: both_synthetics_headers(synthetics_payload, info_payload)) do
+        txn = NewRelic::Agent::Tracer.current_transaction
+
+        assert_equal @last_encoded_header, txn.raw_synthetics_header
+        assert_equal synthetics_payload, txn.synthetics_payload
+        assert_equal info_payload, txn.raw_synthetics_info_header
+        assert_equal expected_info, txn.synthetics_info_payload
+      end
+    end
+
+    def test_load_json
+      info_payload = <<~PAYLOAD
+        {
+          "version": "1",
+          "type": "automatedTest",
+          "initiator": "cli",
+          "attributes": {
+            "attribute1": "on0e"
```

Comment:
Maybe a typo?

```suggestion
            "attribute1": "one"
```

Reply:
fixed this in https://github.com/newrelic/newrelic-ruby-agent/pull/2203/commits/99d2734be84dbdb47b73d626980f4831432b48e3